build/shade/_dotnet-restore.shade

```diff
@@ -10,6 +10,20 @@ restoreDir=''
 default currentDir = '${ Directory.GetCurrentDirectory() }'
 default restoreDir = '${ currentDir }'
 
-default restore_options=' ${ E("KOREBUILD_DOTNET_RESTORE_NO_INFER_RUNTIMES") == "1" ? "" : "--infer-runtimes" } ${ E("KOREBUILD_DOTNET_RESTORE_OPTIONS") }'
```

Comment:
Is someone using `KOREBUILD_DOTNET_RESTORE_NO_INFER_RUNTIMES` today? Will something break?